include/cpp-statsd-client/StatsdClient.hpp

```diff
@@ -130,18 +134,22 @@ class StatsdClient {
     // @{
 
     //! Send a value for a key, according to its type, at a given frequency
-    void send(const std::string& key,
-              const int value,
-              const char* type,
+    template <typename T>
+    void send(const std::string &key,
+              const T value,
+              const char *type,
               float frequency,
-              const std::vector<std::string>& tags) const noexcept;
-
+              const std::vector<std::string> &tags,
+              unsigned int precision = 0) const noexcept;
```

Comment:
precision is a member variable so you dont need to change this method signature, the method has already access to the member variable